docs/api/cases/cases-api-update.asciidoc

```diff
@@ -0,0 +1,235 @@
+[[cases-api-update]]
+== Update cases API
+++++
+<titleabbrev>Update cases</titleabbrev>
+++++
+
+Updates one or more cases.
+
+=== Request
+
+`PATCH <kibana host>:<port>/api/cases`
```

Comment:
We should also document the path that takes a space ID, and document that if the space is not provided in the URL, the default space is used.

```
PATCH <kibana host>:<port>/s/<space_id>/api/cases
```

Reply:
Thanks!  Done in 7a471c994c1997f5ea3284dbd659d669c9ab66b3